packages/storage-plus/src/snapshot/mod.rs

```diff
@@ -10,6 +10,9 @@ use cosmwasm_std::{Order, StdError, StdResult, Storage};
 use serde::de::DeserializeOwned;
 use serde::{Deserialize, Serialize};
 
+/// Structure holding a map of checkpoints composited from
+/// height as U64Key and checkpoint id.
```

Comment:
Is this correct? I have hard time understanding purpose of this `u32`.

Reply:
u32 is the data stored there.

It is a counter of how many times it has been checkpointed (maybe 2 users checkpointed, 1 released, but this stays checkpointed).

Good to add that as a comment

Reply:
I've added that info as comment.